geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako

```diff
@@ -4,45 +4,54 @@ version: '2'
 services:
   db:
     image: ${docker_base}-testdb:${docker_tag}
+% if 'environment' in docker_services.get('db', {}):
```

Comment:
Does such a cooking implies that a full build has to be triggered every time we change db user password ?

Reply:
No It's used only in the docker compose, but if we run `./docker-run make build`, a big part of the build is done...